src/layman/layer/qgis/util_test.py

```diff
@@ -89,9 +89,26 @@ def get_qml_style_path(style_name):
     ('sample/style/small_layer.qml', 'Polygon'),
     ('sample/style/labels_without_symbols/KATASTRALNI_UZEMI_P.qml', 'Polygon'),
     ('sample/style/labels_without_symbols/KATASTRALNI_UZEMI_L.qml', 'Line'),
+    ('sample/style/labels_without_symbols/BODOVE_POLE_T.qml', 'Unknown'),
+    ('sample/style/labels_without_symbols/DALSI_PRVKY_MAPY_T.qml', 'Unknown'),
+    ('sample/style/labels_without_symbols/KATASTRALNI_UZEMI_DEF.qml', 'Unknown'),
+    ('sample/style/labels_without_symbols/PRVKY_ORIENT_MAPY_T.qml', 'Unknown'),
 ])
-def test_get_qml_geometry_from_qml(qml_path, exp_qml_type):
+def test_get_qml_geometry_only_from_qml(qml_path, exp_qml_type):
     parser = ET.XMLParser(remove_blank_text=True)
     qml_xml = ET.parse(qml_path, parser=parser)
-    result = util.get_qml_geometry_from_qml(qml_xml)
+    result = util.get_qml_geometry_only_from_qml(qml_xml)
+    assert result == exp_qml_type
+
+
+@pytest.mark.parametrize('qml_path, db_types, exp_qml_type', [
+    ('sample/style/small_layer.qml', None, 'Polygon'),
+    ('sample/style/labels_without_symbols/KATASTRALNI_UZEMI_P.qml', None, 'Polygon'),
+    ('sample/style/labels_without_symbols/KATASTRALNI_UZEMI_L.qml', None, 'Line'),
+    ('sample/style/labels_without_symbols/BODOVE_POLE_T.qml', ["ST_Point"], 'Point'),
```

Comment:
Test that `('sample/style/labels_without_symbols/BODOVE_POLE_T.qml', ["ST_Polygon"])` raises